packages/react-data-grid/src/common/editors/EditorContainer.tsx

```diff
@@ -256,42 +258,33 @@ export default class EditorContainer extends React.Component {
 
   isCaretAtBeginningOfInput = () => {
     const inputNode = this.getInputNode();
-    return inputNode.selectionStart === inputNode.selectionEnd
-      && inputNode.selectionStart === 0;
+    if (inputNode instanceof HTMLInputElement) {
+      return inputNode.selectionStart === inputNode.selectionEnd
+        && inputNode.selectionStart === 0;
+    }
+    return false;
```

Comment:
Can be simplified
```ts
return inputNode instanceof HTMLInputElement
  && inputNode.selectionEnd === 0
  && inputNode.selectionStart === 0;
```
I don't think we even need the `selectionStart` comparison, as it shouldn't ever be greater than `selectionEnd`.